app/views/pages/help/_other.html.erb

```diff
@@ -3,7 +3,7 @@
 <ul class="features">
   <li><%= link_to t("pages.help.other.how_to_use", org_name: setting['org_name']), how_to_use_path %></li>
 
-  <% SiteCustomization::Page.with_more_info_flag.with_same_locale.each do |custom_page| %>
+  <% SiteCustomization::Page.with_more_info_flag.each do |custom_page| %>
```

Comment:
This one is tricky. IMHO we should only show pages which have a translation for the current locale. How about keeping this line the way it was and changing the scope to:
```ruby
   scope :with_same_locale, -> { joins(:translations).where("site_customization_page_translations.locale": I18n.locale) }
```

Feel free to simplify the condition! This is only an example.

Reply:
I am agree. Will add again with_same_local scope and check to simplify the condition.